tests/client/amo/components/TestAddonReviewList.js

```diff
@@ -164,6 +164,7 @@ describe('amo/components/AddonReviewList', () => {
           assert.ok(dispatch.called);
           const expectedAction = setAddonReviews({ addonSlug, reviews });
           assert.deepEqual(dispatch.firstCall.args[0], expectedAction);
+          return true;
```

Comment:
returning `true` is going to get really annoying. I say we remove this rule. Instead of extending `plugin:promise/recommended` can you just add what we need?